wishlist.html

```diff
@@ -0,0 +1,252 @@
+            alt="Nature"
+            class="img-responsive"
+          />
+        </div>
+        <i class="fas fa-heart card-favourite-icon"></i>
+
+        <div class="card-details">
+          <div class="card-title text-bold text-medium">Denim Mens Jacket</div>
+          <div class="card-category text-bold text-medium">Jacket</div>
+          <div class="card-description">
+            <div class="text-small text-semibold">
+              Sleeveless Colorblock Men Quilted Jacket
+            </div>
+          </div>
+          <div class="card-price text-medium text-semibold">Rs. 5700</div>
+          <button class="btn secondary-btn-outline btn-disabled">
+            Out of stock
+          </button>
+        </div>
+      </div>
+
+      <div class="card-vertical">
+        <div class="card-img">
+          <img
+            loading="lazy"
+            src="./assets/product-image3-landscape.jpeg"
+            alt="Nature"
+            class="img-responsive"
+          />
+        </div>
+        <i class="fas fa-heart card-favourite-icon"></i>
+        <span class="card-badge text-semibold text-center"> New</span>
+
+        <div class="card-details">
+          <div class="card-title text-bold text-medium">Denim Mens Jacket</div>
+          <div class="card-category text-bold text-medium">Jacket</div>
+          <div class="card-description">
+            <div class="text-small text-semibold">
+              Sleeveless Colorblock Men Quilted Jacket
+            </div>
+          </div>
+          <div class="card-price text-medium text-semibold">Rs. 5700</div>
+          <button class="btn secondary-btn">Move to cart</button>
+        </div>
+      </div>
+
+      <div class="card-vertical">
+        <div class="card-img">
+          <img
+            loading="lazy"
+            src="./assets/product-image3-landscape.jpeg"
+            alt="Nature"
+            class="img-responsive"
+          />
+        </div>
+        <i class="fas fa-heart card-favourite-icon"></i>
+
+        <div class="card-details">
+          <div class="card-title text-bold text-medium">Denim Mens Jacket</div>
+          <div class="card-category text-bold text-medium">Jacket</div>
+          <div class="card-description">
+            <div class="text-small text-semibold">
+              Sleeveless Colorblock Men Quilted Jacket
+            </div>
+          </div>
+          <div class="card-price text-medium text-semibold">Rs. 5700</div>
+          <button class="btn secondary-btn">Move to cart</button>
+        </div>
+      </div>
+
+      <div class="card-vertical">
+        <div class="card-img">
+          <img
+            loading="lazy"
+            src="./assets/product-image3-landscape.jpeg"
+            alt="Nature"
+            class="img-responsive"
+          />
+        </div>
+        <i class="fas fa-heart card-favourite-icon"></i>
+        <span class="card-badge text-semibold text-center"> Sold out</span>
```

Comment:
in the sold out card , there should an overlay on the card , so the user can't click on the product.

Reply:
Ok, later i will do it